doc/en/Tutorial - Code a module.rst

```diff
@@ -1488,7 +1493,7 @@ deployment), we have to improve our file ``beavers/populate.py``: ::
     def populate(self):
         [...]
 
-        already_populated = Status.objects.exists()
+        already_populated = BrickDetailviewLocation.objects.filter(brick_id="block_beavers-beaver_age").exists()
```

Comment:
There is now straightforward way to know if the populate script has already be run (one day maybe).
Using 'Status' is better because we create instances which cannot be deleted ('is_custom=False'). With your code if the user remove the brick from its configuration the section of populate which should be executed only the first time could be executed again.